adldap_connector.py

```diff
@@ -537,6 +492,51 @@ def _handle_set_attribute(self, param):
         self.debug_print("[DEBUG] resp = {}".format(self._ldap_connection.response_to_json()))
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_rename_object(self, param):
+        action_result = self.add_action_result(ActionResult(dict(param)))
+        summary = action_result.update_summary({})
+
+        ar_data = {}
+        user = param["object"].lower()
+        new_name = param["new_name"]
+
+        if not self._ldap_bind(action_result):
+            action_result.add_data({"message": "Failed"})
+            summary["message"] = "Failed"
+            return action_result.get_status()
+
+        if param.get("use_samaccountname", False):
+            ret_val, user_dn = self._sam_to_dn([user], action_result=action_result)
+            if phantom.is_fail(ret_val):
+                return action_result.get_status()
+
+            if not user_dn or user_dn.get(user) is False:
+                action_result.add_data({"message": "Failed"})
+                summary["message"] = "Failed"
+                return action_result.set_status(phantom.APP_ERROR, "No users found")
+
+            ar_data["user_dn"] = user_dn[user]
+            ar_data["samaccountname"] = user
+            user = user_dn[user]
+        else:
+            ar_data["user_dn"] = user
+
+        try:
+            self.debug_print("[DEBUG] rename distinguishedName {} to {} ".format(user, new_name))
```

Comment:
I don't think we need to add `[DEBUG]` since the log level is already debug

Reply:
done